.release/update_versions.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+VERSION=$(cat version.txt)
+# Update parent pom version
+mvn versions:set -DnewVersion=$VERSION
+mvn -N versions:update-child-modules
```

Comment:
If I understand this [SO post](https://stackoverflow.com/a/10582306/9414803) correctly:
1. If versions aren't specified in a child pom, it inherits from the parent pom
2. `mvn versions:set` seems to update the parent version in child poms.

Would it be better to remove the explicit child versions, and just run the one command?

Reply:
It looks like you can also use `-DgenerateBackupPoms=false` to avoid cleaning these up

Reply:
Addressed both of these things in the last commit